examples/nats-jetstream-pub/nats_jetstream_pub.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/http;
+import ballerinax/nats;
+
+type Order readonly & record {
+    int orderId;
+    string productName;
+    decimal price;
+    boolean isValid;
+};
+
+service / on new http:Listener(9092) {
+    private final string SUBJECT_NAME = "orders";
+    private final nats:JetStreamClient orderClient;
+
+    function init() returns error? {
+        // Initiate the NATS JetStreamClient at the start of the service. This will be used
+        // throughout the lifetime of the service.
+        self.orderClient = check new (check new nats:Client(nats:DEFAULT_URL));
```

Comment:
Shouldn't we explain the client initialization needs a `nats:Client`? Maybe we can initialize the NATS client in a separate line to make it clear.